actions/receive/action.yaml

```diff
@@ -18,10 +18,10 @@ inputs:
     description: 'Integer to use as cache version. Increment to use new cache.'
     required: true
     default: 1
-  touchstone_ref:
-    description: 'Which branch or tag of {touchstone} should be used. Mainly for debugging.'
```

Comment:
This way it's a breaking change, why not add the _package input and also keep _ref and do this: ` ${{ inputs.touchstone_package }} ${{ inputs.touchstone_ref }}`

That is backwards compatible (there are some users after all^^) but also allows for use of a custom package.

Reply:
Done, thanks!